src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -194,18 +194,23 @@ public Func<EndpointInfo> OtlpServiceHttpEndPointAccessor
 
         _logger = GetLogger();
 
-        // this needs to be explicitly enumerated for each supported language
-        // our language list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
-        var supportedLanguages = new[]
+        // our localization list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
+        var localizedLanguages = new[]
```

Comment:
nit: use collection initializer to create localizedLanguages collection. Potentially a little more efficient than an extension method on array.